Docs changes for Geo-Distributed Placement (#9027)
yugabyte/yugabyte-db: docs/content/latest/explore/ysql-language-features/partitions.md

Partitioning ranges are inclusive at the lower ( `FROM` ) bound and exclusive at the upper ( `TO` ) bound.
Each month range in the preceding examples includes the start of the month, but does not include the start of the following month.

> If you wish to create a new partition that merely contains all the rows that do not match the specified partitions,
> you can do so by adding a default partition in the following manner:

Reviewer: ```suggestion
To create a new partition that contains only the rows that don't match the specified partitions, add a default partition as follows:
```
